spellbook/models/ens/ens_view_registries.sql

```diff
@@ -0,0 +1,14 @@
+{{config(alias='view_registries')}}
+SELECT 
+  node,
+  label,
+  min(evt_block_time) AS min_evt_block_time, 
+  max(evt_block_time) AS max_evt_block_time, 
+  count(*)  
```

Comment:
This one is a bit weird, but basically it fails because count(*) needs an alias (e.g., count(*) as count_ens_registries)